linux/jdk/debian/src/main/packaging/microsoft/11/debian/control

```diff
@@ -0,0 +1,62 @@
+Source: msopenjdk-11
+Section: java
+Priority: optional
+Maintainer: Microsoft Package Maintainers <openjdk@microsoft.com>
+Build-Depends: debhelper (>= 11), lsb-release
+
+Package: msopenjdk-11
+Architecture: amd64 arm64
+Depends: adoptium-ca-certificates,
+         java-common,
+         libasound2,
+         libc6,
+         libx11-6,
+         libfontconfig1,
+         libfreetype6,
+         libxext6,
+         libxi6,
+         libxrender1,
+         libxtst6,
+         zlib1g
+Recommends: fonts-dejavu-core,
+            fonts-dejavu-extra
+Provides: java-compiler,
+          java-sdk,
+          java-sdk-headless,
+          java10-sdk,
+          java11-sdk,
+          java2-sdk,
+          java5-sdk,
+          java6-sdk,
+          java7-sdk,
+          java8-sdk,
+          java9-sdk,
+          java10-sdk-headless,
+          java11-sdk-headless,
+          java2-sdk-headless,
+          java5-sdk-headless,
+          java6-sdk-headless,
+          java7-sdk-headless,
+          java8-sdk-headless,
+          java9-sdk-headless,
+          java-runtime,
+          java-runtime-headless,
+          java10-runtime,
+          java11-runtime,
+          java2-runtime,
+          java5-runtime,
+          java6-runtime,
+          java7-runtime,
+          java8-runtime,
+          java9-runtime,
+          java10-runtime-headless,
+          java11-runtime-headless,
+          java2-runtime-headless,
+          java5-runtime-headless,
+          java6-runtime-headless,
+          java7-runtime-headless,
+          java8-runtime-headless,
+          java9-runtime-headless
+Description: Microsoft Build of OpenJDK 11
+ Microsoft Build of OpenJDK is an OpenJDK-based development environment to create
+ applications and components using the programming language Java.
```

Comment:
```suggestion
 applications and components using the Java programming language.
```

Reply:
If accepted then apply this change for 17 also.

Reply:
Accepted 👍